sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CognitiveComplexityWalker.cs

```diff
@@ -51,6 +53,25 @@ public void EnsureVisitEndedCorrectly()
             }
         }
 
+        public void Walk(SyntaxNode node)
+        {
+            try
+            {
+                Visit(node);
+            }
+            catch (InsufficientExecutionStackException)
+            {
+                // TODO: trace this exception

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)